docs/Installation/Installation.md

```diff
@@ -53,14 +53,14 @@ all of these dependencies.
 * [h5py](https://www.h5py.org)
 
 #### Optional:
-* [Pybind11](https://pybind11.readthedocs.io) 2.6.0 or later for SpECTRE Python
+* [Pybind11](https://pybind11.readthedocs.io) 2.6.0 or later — for SpECTRE Python
```

Comment:
I don't think this is a maintainable change. People will just use the `-` key on their keyboard. I think you're trying to fight a losing battle. I think it's better to just drop the commit. It tex code where it's easy to do it's good to be correct, just I think here it's not maintainable.

Reply:
Should I flip the commit around then? Make all the em dashes into hyphens for consistency?
As a reader, I do prefer the em dashes because they separate the text a little more and it's easier to read, but I agree that it can't be maintained (I didn't even notice the first time).
In that case I'd still like things to be consistent.

Reply:
I would just drop the em dash commit completely. Next time anybody else adds something it will be inconsistent. I don't disagree it looks nicer, the maintenance overhead is just not worth it in my opinion.